src/scripts/main.js

```diff
@@ -1,3 +1,442 @@
+  }
+}
+
+function errorPosition(e, userForm) {
+  if (userForm['position'].value.length === 0) {
+    position.style.display = 'block';
+
+    setTimeout(() => {
+      position.style.display = 'none';
+    }, 2000);
+  } else {
+    errorAge(e, userForm);
+  }
+}
+
+function errorAge(e, userForm) {
+  if (userForm['age'].value === '') {
+    invalidAge.style.display = 'block';
+
+    setTimeout(() => {
+      invalidAge.style.display = 'none';
+    }, 2000);
+  } else {
+    if (userForm['age'].value < 18) {
+      youngAge.style.display = 'block';
+
+      setTimeout(() => {
+        youngAge.style.display = 'none';
+      }, 2000);
+    } else {
+      errorSalary(e, userForm);
+    }
+  }
+}
+
+function errorSalary(e, userForm) {
+  if (userForm['salary'].value.length === 0) {
+    salary.style.display = 'block';
+
+    setTimeout(() => {
+      salary.style.display = 'none';
+    }, 2000);
+  } else {
+    succeffully(e, userForm);
+  }
+}
+
+function succeffully(e, userForm) {
+  success.style.display = 'block';
+
+  setTimeout(() => {
+    success.style.display = 'none';
+  }, 2000);
+
+  addPerson(e, userForm);
+}
+
+function addPerson(e, userForm) {
+  data.push({
+    isActive: false,
+    name: userForm['name'].value,
+    position: userForm['position'].value,
+    office: userForm['office'].value,
+    age: userForm['age'].value,
+    money: +userForm['salary'].value.replace(/\D/g, ''),
+  });
+
+  setTimeout(() => {
+    addEventTocell();
+  }, 500);
+
+  createHTML(e);
+}
+
+function changeValue(e) {
+  removeEventTocell();
+
+  const indexOf = e.currentTarget.innerHTML.indexOf('$');
+
+  oldValue = indexOf === 0
+    ? +e.currentTarget.innerHTML.replace(/\D/g, '').toLocaleString('en')
+    : e.currentTarget.innerHTML;
+
+  newValue = indexOf === 0
+    ? +e.currentTarget.innerHTML.replace(/\D/g, '').toLocaleString('en')
+    : e.currentTarget.innerHTML;
+  selectedCell = e.currentTarget;
+
+  newInput.addEventListener('keyup', () => {
+    writeNewValue(indexOf);
+  });
+
+  e.currentTarget.innerHTML = '';
+  newInput.value = oldValue;
+  selectedCell.append(newInput);
+  newInput.focus();
+
+  newInput.addEventListener('blur', () => {
+    addNewValue(indexOf);
+  });
+}
+
+function addNewValue(indexOf) {
+  setTimeout(() => {
+    if (newValue === '' || newValue === 0) {
+      selectedCell.innerHTML
+      = indexOf === 0 ? `$${oldValue.toLocaleString('en')}` : oldValue;
+    } else {
+      setTimeout(() => {
+        selectedCell.innerHTML
+        = indexOf === 0 ? `$${newValue.toLocaleString('en')}` : newValue;
+        newInput.remove();
+      }, 10);
+    }
+  },);
+
+  for (let i = 0; i < tbody.querySelectorAll('td').length; i++) {
+    tbody.querySelectorAll('td')[i].addEventListener('click', changeValue);
+  }
+}
+
+function writeNewValue(indexOf) {
+  if (event.code === 'Enter') {
+    addNewValue(indexOf);
+  }
+
+  newValue = indexOf === 0
+    ? +newInput.value.replace(/\D/g, '') : newInput.value;
```

Comment:
зазвича в тернарному операторі переносять на новий рядок починаючи з ключових символів
```suggestion
  newValue = indexOf === 0
    ? +newInput.value.replace(/\D/g, '') 
    : newInput.value;
```

Reply:
так легше читати